src/org/zaproxy/zap/extension/exportreport/resources/help/contents/exportreport.html

```diff
@@ -1,334 +1,377 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"> 
 <HTML>
-	<HEAD>
-		<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
-		<TITLE>
-			Export Report
-		</TITLE>
-	</HEAD>
+    <HEAD>
+        <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+        <TITLE>
+            Export Report
+        </TITLE>
+    </HEAD>
 
-	<BODY BGCOLOR="#ffffff">
+    <BODY BGCOLOR="#ffffff">
 
-		<H1><U>Export Report Version 1.0</U></H1>
-		Report Export module that allows users to customize content and export in a desired format.
-		<UL>
-			<LI>Created by <A href="https://github.com/JordanGS">JordanGS</A>.</LI>
-			<LI>Minimum Supported Version: Weekly Release ZAP_D-2016-08-15</LI>
-			<LI>Project on  <A href="https://github.com/JordanGS/workspace">GitHub</A>. You can also find a zip of the minimum supported release in the zap-download folder.</LI>
+        <H1><U>Export Report Version 2</U></H1>
+        Report Export module that allows users to customize content and export in a desired format.
+        <UL>
+            <LI>Created by <A href="https://github.com/JordanGS">JordanGS</A>.</LI>
+            <LI>Minimum Supported Version: Weekly Release ZAP_D-2016-09-05</LI>
```

Comment:
@thc202 meet me in the middle here because it doesn't generate anything in 2.5 with this plugin or the default generation :(

The jenkins plugin has the same problem : / Once 2.6 is released, i'll gladly update both :) but please let me test 2.6 before release to make sure it generates reports :)